src/components/FlexibleForm/Control.js

```diff
@@ -208,6 +210,16 @@ export const Control = memo(props => {
         wrapper = decorators[decorator];
       }
 
+      if(decorator && decorator === 'withBooleanActions' && staticControlType === 'KeyValue') {
+        const actions = get(FORMS_SETTINGS, [ENTITY_KEYS.MAPPING_PROFILES, 'DECORATORS', 'BOOLEAN_ACTIONS'], []);
+        const newValue = actions.find(item => item.value === attrs?.value)?.label;
+
+        attrs = {
+          ...attrs,
+          value: newValue ? (<FormattedMessage id={newValue} />) : newValue,
+        };
+      }
+
```

Comment:
I think it should be inside your decorator, isn't it?

Reply:
No, It used for ViewDetails section in static view mode, and decorator knows nothing about Component, which renders in staticView mode